### PR TITLE
BZ1932466: added missing namespace for PVC

### PR DESCRIPTION
**Pull request by chinmayi-chandrasekar**
Applicable for 4.5+

https://bugzilla.redhat.com/show_bug.cgi?id=1932466

Doc preview link: https://deploy-preview-34315--osdocs.netlify.app/openshift-enterprise/latest/registry/configuring_registry_storage/configuring-registry-storage-vsphere.html#installation-registry-storage-block-recreate-rollout_configuring-registry-storage-vsphere

Requires QA approval from @RazTamir

### CODE REVIEW

File: modules/installation-registry-storage-block-recreate-rollout.adoc [hunk 1]
```diff
@@ -45,16 +45,18 @@ kind: PersistentVolumeClaim
...
   resources:
     requests:
-      storage: 100Gi <3>
+      storage: 100Gi <4>
 ----
 <1> A unique name that represents the `PersistentVolumeClaim` object.
-<2> The access mode of the PersistentVolumeClaim. With `ReadWriteOnce`, the volume can be mounted with read and write permissions by a single node.
-<3> The size of the PersistentVolumeClaim.
+<2> The namespace for the `PersistentVolumeClaim` object, which is `openshift-image-registry`.
+<3> The access mode of the PersistentVolumeClaim. With `ReadWriteOnce`, the volume can be mounted with read and write permissions by a single node.
```
Comment (mburke5678) [inline modules/installation-registry-storage-block-recreate-rollout.adoc:58]: s/PersistentVolumeClaim/persistent volume claim when not followed by _object_.

File: modules/installation-registry-storage-block-recreate-rollout.adoc [hunk 2]
```diff
@@ -45,16 +45,18 @@ kind: PersistentVolumeClaim
...
+  - ReadWriteOnce <3>
   resources:
     requests:
-      storage: 100Gi <3>
+      storage: 100Gi <4>
 ----
 <1> A unique name that represents the `PersistentVolumeClaim` object.
-<2> The access mode of the PersistentVolumeClaim. With `ReadWriteOnce`, the volume can be mounted with read and write permissions by a single node.
-<3> The size of the PersistentVolumeClaim.
+<2> The namespace for the `PersistentVolumeClaim` object, which is `openshift-image-registry`.
```
Comment (mburke5678) [inline modules/installation-registry-storage-block-recreate-rollout.adoc:57]: Is the project for a PVC always `openshift-image-registry`, or just in this example?

File: modules/installation-registry-storage-block-recreate-rollout.adoc [hunk 3]
```diff
@@ -45,16 +45,18 @@ kind: PersistentVolumeClaim
...
+<3> The access mode of the PersistentVolumeClaim. With `ReadWriteOnce`, the volume can be mounted with read and write permissions by a single node.
+<4> The size of the PersistentVolumeClaim.
```
Comment (mburke5678) [inline modules/installation-registry-storage-block-recreate-rollout.adoc:59]: s/PersistentVolumeClaim/persistent volume claim

**Comment by mburke5678** [comment]
@chinmayi-chandrasekar I have a few nits. Otherwise LGTM.

File: modules/installation-registry-storage-block-recreate-rollout.adoc (hunk 2 above)
Comment (chinmayi-chandrasekar) [reply modules/installation-registry-storage-block-recreate-rollout.adoc:57]: Yes, it should always be configured for openshift-image-registry (as per the bug description).

**Comment by vikram-redhat** [comment]
@jhou1 Hi - are you able to validate this PR?

**Review by xgu-redhat — APPROVED** [review]
/lgtm

**Review by xgu-redhat — APPROVED** [review]
/lgtm

**Review by xgu-redhat — APPROVED** [review]
/lgtm

**Review by kalexand-rh — COMMENTED** [review]
LGTM!

**Pull request merged**